src/modules/party/party.service.test.ts

```diff
@@ -65,6 +84,86 @@ describe('PartyService', () => {
     },
   ];
 
+  const createPartyRequest: CreatePartyRequestItem = {
+    alternateIdentifier: '00309999',
+    industryClassification: '2401',
+    name1: 'Longer than 20 characters for testing purposes',
+    name2: '',
+    name3: '',
+    smeType: '20',
+    citizenshipClass: '2',
+    officerRiskDate: '2018-03-21',
+    countryCode: 'DZA',
+  };
+
+  const acbsCreatePartyRequest: AcbsCreatePartyRequest = {
```

Comment:
Should these be loaded from mock requests, I think it will be beneficial to have a centralised mocked requests for myriads of requests where some will overlap?

Reply:
Just to clarify, do you mean using a generator class to generate these mock requests (e.g. `get-deal-investors-by-deal-identifier.api-test.ts` uses `DealInvestorGenerator`)?

Reply:
As discussed after stand-up, we could either take the approach of having constants for mock requests or using generator classes. I decided to go with a generator class, adding `CreatePartyGenerator`.